src/assets/i18n/pl.json

```diff
@@ -5,11 +5,17 @@
     "SIGN_OUT_ERROR": "Wystapił błąd podczas wylogowania"
   },
   "COMMON": {
+    "APPLE": "Jabłko",
+    "BREAD": "Pieczywo",
+    "CARROT": "Marchew",
     "EMAIL": "Email",
+    "MILK": "Mleko",
     "NAME": "Nazwa",
     "INGREDIENTS": "Składniki",
-    "QUANTITY": "Ilość",
     "PASSWORD": "Hasło",
+    "RICE": "Ryż",
+    "SALAD": "Sałata",
+    "QUANTITY": "Ilość",
     "DAY": {
```

Comment:
Jak mamy już realne elementu w tych obiektach to potrzebujemy te tłumaczenia?

Reply:
faktycznie nie :D